### PR TITLE
Display product reference in Stock management list (ASM)

### PR DESCRIPTION
**Pull request by hibatallahAouadni**
| Questions     | Answers
| ------------- | -------------------------------------------------------
| Branch?       | 1.6.1.x
| Description?  | in BO, Stock -> Stock management, the references, ean13 and upc of products with combinations doesn't show
| Type?         | bug fix
| Category?     | BO
| BC breaks?    | no
| Deprecations? | no
| Fixed ticket? | http://forge.prestashop.com/browse/PSCSX-7742
| How to test?  | add some products with combinations, make sure to put a reference, an ean13 and an upc for the product and to mange it with ASM, then go to Stock -> Stock management, you will see the list of products with references, ean13 and upc shown

### CODE REVIEW

**Comment by marionf** [comment]
Hello @hibatallahAouadni 

capture du 2017-06-12 17-44-12
capture du 2017-06-12 17-43-51

Can we also solve the problem with the EAN13 and UPC code ?

**Comment by hibatallahAouadni** [comment]
@marionf sure with pleasure ;)

**Comment by hibatallahAouadni** [comment]
@marionf done :+1:  and I added some tests if one of this field is empty then '--' will be displayed instead

File: controllers/admin/AdminStockManagementController.php [hunk 1]
```diff
@@ -221,6 +221,9 @@ public function getList($id_lang, $order_by = null, $order_way = null, $start =
         for ($i = 0; $i < $nb_items; $i++) {
             $item = &$this->_list[$i];
 
+            $item['reference'] = (!empty($item['reference']) && isset($item['reference'])) ? $item['reference'] : '--';
+            $item['ean13'] = (!empty($item['ean13']) && isset($item['ean13'])) ? $item['ean13'] : '--';
+            $item['upc'] = (!empty($item['upc']) && isset($item['upc'])) ? $item['upc'] : '--';
```
Comment (hibatallahAouadni) [inline controllers/admin/AdminStockManagementController.php]: @marionf here's the tests to check if one of these fields is empty

Comment (marionf) [inline controllers/admin/AdminStockManagementController.php]: Thank you @hibatallahAouadni :+1:

**Comment by maximebiloe** [comment]
Thanks @hibatallahAouadni

**Pull request merged**